src/lib/slack.ts

```diff
@@ -34,3 +34,34 @@ export async function postToSlack(
     return Promise.reject(e)
   }
 }
+
+export type PostFeedbackToSlackOptions = {
+  attachments: MessageAttachment[]
+  webhookUrl: string
+  username?: string
+  text?: string
+}
+
+export async function postFeedbackToSlack(
+  options: PostFeedbackToSlackOptions,
+): Promise<ChatPostMessageResponse> {
+  const {webhookUrl, text, attachments, username = 'Eggo'} = options
+  try {
+    await fetch(webhookUrl, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        text,
+        attachments,
+        username,
+      }),
+    })
```

Comment:
the slack token we have set up doesn't have access to the `egghead-feedback` channel. Sussing out permissions for slack apps has been a huge pain so this sends the feedback to their `Incoming Webhook` url that you can set up on a per-channel basis